server/controllers/categories/getAllCategories.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Response, Request } from 'express';
+import { Category } from '../../database/models';
+import { CustomedError } from '../../utils';
+
+const getAllCategories = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const categories: any = await Category.findAll({
+      attributes: ['id', 'name', 'icon_url'],
```

Comment:
I think these are all the attributes in the categories table, right? . If yes then you dont need to specified the attributes